### DISCUSSION TITLE
How we can define several IP segments as "EXTERNAL_NET"?

### DISCUSSION THREAD

**Discussion by kornflake003-GITHUB** [Security-Onion-Solutions/securityonion#6449] [Unsupported Versions] [answered]
Hi,

SOC evaluation versión:

How we can define several IP segments as "EXTERNAL_NET"?. Default configuration are define like "any". There is a variable like "hnmanager" in global.sls to set a diferent IP segments in EXTERNAL_NET?

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/suricata.html?highlight=external_net#configuration
